.gitlab-ci.yml

```
@@ -0,0 +1,105 @@
+stages:
+  - test
+  
+include:
+  - project: viz/ci/gitlabpipelines
+    file: index.yml
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
```

I would add this here (as default/tags is set specifically by `hpc/gitlab-pipelines`, this needs to be overriden)
```suggestion

default:
  tags: []
```